### PR TITLE
Fixed 2 Typos in the Documentation

### PR DESCRIPTION
**Pull request by kritikush**
"cam" -> "can"
"mmapped into" -> "mapped to"

### CODE REVIEW

**Comment by kritikush** [comment]
@googlebot I signed it!

File: examples/colab/semantic_approximate_nearest_neighbors.ipynb [hunk 1]
```diff
@@ -655,7 +655,7 @@
       "source": [
         "## 3. Build the ANN Index for the Embeddings\n",
         "\n",
-        "[ANNOY](https://github.com/spotify/annoy) (Approximate Nearest Neighbors Oh Yeah) is a C++ library with Python bindings to search for points in space that are close to a given query point. It also creates large read-only file-based data structures that are mmapped into memory. It is built and used by [Spotify](https://www.spotify.com) for music recommendations."
```
Comment (arnoegw) [inline examples/colab/semantic_approximate_nearest_neighbors.ipynb:658]: This is not a typo but correct techspeak, see https://en.wikipedia.org/wiki/Mmap. Please leave the text as-is. Feel free to add an explanatory link as seen at https://github.com/spotify/annoy/blob/df5364c9b39526bb7b53e31b1b5fd593e879b58a/README.rst#annoy

**Review by arnoegw — COMMENTED** [review]
Thank you for contributing!

File: examples/colab/semantic_approximate_nearest_neighbors.ipynb (hunk 1 above)
Comment (kritikush) [reply examples/colab/semantic_approximate_nearest_neighbors.ipynb:658]: Got it !

**Comment by arnoegw** [comment]
Oops, I see you closed this. But the "cam"->"can" fix was actually very welcome. I've applied it now in commit https://github.com/tensorflow/hub/commit/3409e89e39ef1156ee4ef8635b5c1f6f0433e2d5 with reference to you.

Thanks again for contributing!

**Comment by kritikush** [comment]
> Oops, I see you closed this. But the "cam"->"can" fix was actually very welcome. I've applied it now in commit [3409e89](https://github.com/tensorflow/hub/commit/3409e89e39ef1156ee4ef8635b5c1f6f0433e2d5) with reference to you.
> 
> Thanks again for contributing!

 Thanks !

**Pull request closed without merging**